Documentation/Blazorise.Docs/Pages/News/2024-02-15-release-notes-150.razor

```diff
@@ -259,4 +259,12 @@
     In this release, we've updated our AntDesign provider. Previously, we were using the older 4.0 version of AntDesign, which didn't do justice to this exceptional framework. Although we aim to upgrade to the latest AntDesign 5.1 version in the upcoming Blazorise milestone, this update addresses several issues identified while developing Blazorise 1.5.
 </Paragraph>
 
+<Heading Size="HeadingSize.Is3">
+    TreeView context menu
+</Heading>
+
+<Paragraph>
+    It is now possible to detect when the user right click on a TreeView node. By using the <Code>NodeContextMenu</Code> event callback you can detect the clicked Node and mouse coordinates tos how your custom context menu.
```

Comment:
typo "tos how"

Reply:
This is just a template for me. I'm already working on rewriting release notes for another branch.